lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -115,6 +115,26 @@ class ReportRenderer {
   }
 
   /**
+   * Returns a div with a list of top-level warnings, or null if no warnings.
+   * @param {!ReportRenderer.ReportJSON} report
+   * @return {?DocumentFragment}
+   */
+  _renderReportWarnings(report) {
+    if (!report.lighthouseRunWarnings || report.lighthouseRunWarnings.length === 0) {
+      return null;
```

Comment:
alternatively this could return an empty div so the caller wouldn't need to check the results

Reply:
I suppose I have a slight preference for empty div, but I don't feel terribly strongly if this is a pattern we can expect to employ elsewhere as well (I can't think of any off the top of my head, but maybe you borrowed from somewhere?)

Reply:
> I have a slight preference for empty div

done